uint8 LEARNING = 0
uint8 RECOGNIZING = 1

uint8 learningMode
